README.md

```diff
@@ -4,6 +4,11 @@ This repo contains analysis notebooks used in the _The Clinical Genomic Variatio
 
 Small output files can be found in this repo. Larger files can be found in our public s3 bucket: `s3://nch-igm-wagner-lab-public/variation-normalizer-manuscript/`. There are notebooks that provide functions for programmatically downloading files from the s3 bucket.
 
+### Results
+![Patient Matching with GENIE](./analysis/genie/variant_analysis/genie_patient_matching.png)
+
+Variant normalization allows patient data from AACR Project GENIE to be matched to normalized variants in the CIViC, MOA, and ClinVar knowledgebases.
+
```

Comment:
I would move this to the beginning like @wesleygoar had mentioned as a possible place